cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -313,6 +365,33 @@ def get_kube_api_server(kube_api):
     return server
 
 
+def kubectl(operation, manifest):
+    ''' Wrap the kubectl creation command when using filepath resources
+    :param operation - one of get, create, delete, replace
+    :param manifest - filepath to the manifest
+     '''
+    kubectl = ['kubectl', '--kubeconfig=/srv/kubernetes/config']
+    # determine if the kubernetes resources have been declared already
+    found = call(kubectl + ['get', '-f', manifest])
+
+    if operation == 'delete':
+        return call(kubectl + [operation, '-f', manifest])
+        hookenv.log('Executed {}'.format(' '.join(kubectl + [operation, '-f',
+                                                  manifest])))
+    else:
+        # If no resource was found, attempt creating it
+        if found != 0:
+            command = kubectl + [operation, '-f', manifest]
+            if operation == 'delete':
```

Comment:
If operation is delete this method has already returned. I *think* this is dead code. also if the operation is not delete no operation is executed.